src/Microsoft.OpenApi/Services/OpenApiVisitorBase.cs

```diff
@@ -44,6 +58,15 @@ public string PathString
             }
         }
 
+        internal void AttachCurrentKeys(CurrentKeys currentKeys)
+        {
+
+        }
+
+        internal void ClearCurrentKeys()
+        {
+
+        }
     
```

Comment:
This is not used, and not marked virtual. Not sure what it is for?

Reply:
I forgot to delete it.  I did have CurrentKeys in the Walker and then I moved it into the visitor base.